pull_request_template.md

```diff
@@ -0,0 +1,87 @@
+<!--Thank you for contributing to AllenSDK, your work and time will help to
+advance open science! For full contribution guidelines check out our
+guide on GitHub here, https://github.com/AllenInstitute/AllenSDK/blob/master/CONTRIBUTING.md-->
+
+# Overview:
+<!-- Give a brief overview of the issue you are solving. Succinctly
+explain the GitHub issue you are addressing and the underlying problem
+of the ticket
+example: 
+Science team is not able to load max or avg projections for experiment
+session #blah. A image cannot be created because input pixel
```

Comment:
instead of `#blah` I would change that to something a bit more formal like `#x`

Reply:
changed